src/diagnostics.rs

```diff
@@ -118,7 +118,12 @@ pub fn report_error<'tcx, 'mir>(
     let result = report_msg(ecx, &format!("{}: {}", title, msg), msg, helps, true);
```

Comment:
`report_msg` always returns `None`, no idea why it even has a return type... we can clean that up later though if you prefer.